readthedocs/source/orb/gettingstarted.md

```diff
@@ -0,0 +1,1205 @@
+      "did:orb:hl:uEiBtg3HASdLSUHLYds6givvkjJX_DEb8cqONqpKuq_Y_aA:uoQ-BeEVodHRwOi8vb3JiMS5sb2NhbC9jYXMvdUVpQnRnM0hBU2RMU1VITFlkczZnaXZ2a2pKWF9ERWI4Y3FPTnFwS3VxX1lfYUE:EiCGIu4PrFTVEZRb8SOZt4nbZRF7Wp2_qX4Zt0czROKHxg"
+    ],
+    "method": {
+      "updateCommitment": "EiDsq7uyQDTYJss8JAV68EcVb1sJHXgUl8knESbzvN4RWw",
+      "recoveryCommitment": "EiAVtjXx-WHYfWcAzzgbHdcADhF1KVSzpy5Tb_xnu1cbzA",
+      "published": true,
+      "anchorOrigin": "http://orb1.local",
+      "publishedOperations": [
+        {
+          "operation": "eyJkZWx0YSI6eyJwYXRjaGVzIjpbeyJhY3Rpb24iOiJhZGQtcHVibGljLWtleXMiLCJwdWJsaWNLZXlzIjpbeyJpZCI6ImtleTIiLCJwdWJsaWNLZXlKd2siOnsiY3J2IjoiUC0yNTYiLCJrdHkiOiJFQyIsIngiOiJiR005YU51ZnBLTlB4bGt5YWNVMWhHaFFYbV9hQzhoSXpTVmVLRHB3akJ3IiwieSI6IlBmZG1DT3RJZFZZMkI2dWNSNG9Ra3Q2ZXZRZGRZaE95SG9EWUNhSTJCSkEifSwicHVycG9zZXMiOlsiYXV0aGVudGljYXRpb24iXSwidHlwZSI6Ikpzb25XZWJLZXkyMDIwIn0seyJpZCI6ImtleTEiLCJwdWJsaWNLZXlKd2siOnsiY3J2IjoiRWQyNTUxOSIsImt0eSI6Ik9LUCIsIngiOiJvMWJHMVU3RzNDTmJ0QUxNYWZVaUZPcThPRHJhVHlWVG1QdFJETzFRVVdnIn0sInB1cnBvc2VzIjpbImF1dGhlbnRpY2F0aW9uIl0sInR5cGUiOiJFZDI1NTE5VmVyaWZpY2F0aW9uS2V5MjAxOCJ9XX0seyJhY3Rpb24iOiJhZGQtc2VydmljZXMiLCJzZXJ2aWNlcyI6W3siaWQiOiJzdmMxIiwicHJpb3JpdHkiOjEsInJlY2lwaWVudEtleXMiOlsia2V5MSJdLCJyb3V0aW5nS2V5cyI6WyJrZXkxIl0sInNlcnZpY2VFbmRwb2ludCI6eyJyb3V0aW5nS2V5cyI6WyJrZXkxIl0sInVyaSI6Imh0dHBzOi8vZXhhbXBsZS5jb20ifSwidHlwZSI6InR5cGUxIn0seyJpZCI6InN2YzIiLCJwcmlvcml0eSI6MiwicmVjaXBpZW50S2V5cyI6WyJrZXkyIl0sInJvdXRpbmdLZXlzIjpbImtleTIiXSwic2VydmljZUVuZHBvaW50Ijp7InJvdXRpbmdLZXlzIjpbImtleTIiXSwidXJpIjoiaHR0cHM6Ly9leGFtcGxlLmNvbSJ9LCJ0eXBlIjoidHlwZTIifV19XSwidXBkYXRlQ29tbWl0bWVudCI6IkVpRHNxN3V5UURUWUpzczhKQVY2OEVjVmIxc0pIWGdVbDhrbkVTYnp2TjRSV3cifSwic3VmZml4RGF0YSI6eyJhbmNob3JPcmlnaW4iOiJodHRwOi8vb3JiMS5sb2NhbCIsImRlbHRhSGFzaCI6IkVpQUE0clRFcTdIUFFIdVNIVGFTSlVmWTFMV2x3cmkzTVlGZUtyYVZBWUN6OUEiLCJyZWNvdmVyeUNvbW1pdG1lbnQiOiJFaUFWdGpYeC1XSFlmV2NBenpnYkhkY0FEaEYxS1ZTenB5NVRiX3hudTFjYnpBIn0sInR5cGUiOiJjcmVhdGUifQ==",
+          "transactionTime": 1653411114,
+          "type": "create",
+          "anchorOrigin": "http://orb1.local",
+          "canonicalReference": "uEiBtg3HASdLSUHLYds6givvkjJX_DEb8cqONqpKuq_Y_aA",
+          "equivalentReferences": [
+            "hl:uEiBtg3HASdLSUHLYds6givvkjJX_DEb8cqONqpKuq_Y_aA:uoQ-BeEVodHRwOi8vb3JiMS5sb2NhbC9jYXMvdUVpQnRnM0hBU2RMU1VITFlkczZnaXZ2a2pKWF9ERWI4Y3FPTnFwS3VxX1lfYUE"
+          ]
+        }
+      ]
+    }
+  }
+}
+```
+
+Get hash of anchor from the _resolve_ response (from the "canonicalId" field in the metadata) and set the ANCHOR_HASH environment variable.
+Verify that the proofs in the anchor linkset are in the VCT log(s):
+
+**_NOTE:_**  Set ANCHOR_HASH to the anchor hash of the DID in the _did resolve_ response.
+
+```commandline
+export ANCHOR_HASH=uEiBtg3HASdLSUHLYds6givvkjJX_DEb8cqONqpKuq_Y_aA
+
+orb-cli vct verify --cas-url=http://orb1.local/cas --anchor=${ANCHOR_HASH}
+```
+
+Response:
+
+```json
+[
+  {
+    "domain": "http://orb.vct:8077/maple2022",
```

Comment:
It would be good to have sample where audit path is not empty (tree size > 1). Or at least mention that for tree size > 1 audit path will be filled in.